### ISSUE TITLE
Code of Conduct Comparison

### ISSUE THREAD

**Issue by dkastl** [pgRouting/.github#1]
Not sure which Code of Conduct is more community-friendly:

- https://github.com/pgRouting/pgrouting/blob/master/CODE_OF_CONDUCT.md
- https://github.com/pgRouting/.github/blob/master/CODE_OF_CONDUCT.md

To me the one proposed by Github looks better.

**Comment by cvvergara** [comment]
On pgRouting we are using OSGeo's code of conduct

**Comment by dkastl** [comment]
Yes, I know. And I see that the OSGeo was made befor the one that Github offers as the default. What I like with the one of Github: it's adopted by lots of projects and is taken from here https://www.contributor-covenant.org/
I also find it more comfortable to read. But I'm OK with any, just wanted to mention the alternative.
Everything in this repository is take as a default but can be overridden by each individual repository.
